src/app/features/global-error/global-error.component.html

```diff
@@ -1,10 +1,12 @@
-<div *ngIf="errorMessage$ | async as errorMessage" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" data-module="govuk-error-summary">
-  <h2 class="govuk-error-summary__title" id="error-summary-title">There is a problem</h2>
-  <div class="govuk-error-summary__body">
-    <ul class="govuk-list govuk-error-summary__list">
-      <li>
-        <a href="#global-error">{{ errorMessage }}</a>
-      </li>
-    </ul>
+<ng-container *ngIf="globalErrorService.errors$ | async as errorMessages">
+  <div *ngIf="errorMessages && errorMessages.length > 0" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" data-module="govuk-error-summary">
+    <h2 class="govuk-error-summary__title" id="error-summary-title">There is a problem</h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <li *ngFor="let error of errorMessages">
+          <a [routerLink]="" queryParamsHandling="preserve" (click)="goto(error)">{{ error.message }}</a>
```

Comment:
Can we include the URL/href here, even though we're using a `(click)` even handler? I only ask in case a screen-reader or some other assistive technology might not perform a click in which case we'd fall-back to the href anchor action

Reply:
I had originally used the fragment directly in the `<a>` but then for errors that don't have a link e.g. API error, there would be no id element look-up

Reply:
I think for API errors we might use the [service error pages](https://design-system.service.gov.uk/patterns/problem-with-the-service-pages/) and redirect (so we ought to have a id element in the situation covered by this PR)

Reply:
We can do this when we introduce the service error page